static/app/components/feedback/list/feedbackListHeader.tsx

```diff
@@ -117,11 +165,17 @@ function HasSelection({
         <ErrorBoundary mini>
           <Button
             onClick={() => {
-              addLoadingMessage(t('Updating feedback...'));
               const newStatus =
                 mailbox === 'resolved' ? GroupStatus.UNRESOLVED : GroupStatus.RESOLVED;
-              resolve(newStatus, mutationOptions);
-              deselectAll();
+              openConfirmModal({
+                onConfirm: () => {
+                  resolve(newStatus, mutationOptionsResolve);
```

Comment:
I'd put the addLoadingMessage() call here, even if it means some duplication, because then it's real close to the success/error messages too.